boards/aitewinrobot-esp32c3-supermini/board.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "Aitewin Robot ESP32-C3 SuperMini",
+  "version": 1.8,
+  "description": "A common ESP32 dev board by AITEWIN ROBOT (Derived from Adafruit QT Py ESP32-C3)",
+  "author": "GeekGarage",
+  "mcu": "esp32-c3",
+  "fqbn": "esp32:esp32:esp32c3",
+  "width": 18.00,
+  "height": 24.576,
+  "pins": {
+    "5V": { "x": 16.702, "y": 3.238, "target": "power(5)" },
+    "GND": { "x": 16.702, "y": 5.778, "target": "GND" },
+    "3V3": { "x": 16.525199, "y": 8.318, "target": "power(3.3)" },
+    "4/GPIO4/A4/SCK": { "x": 16.702, "y": 10.858, "target": "GPIO4" },
+    "3/GPIO3/A3": { "x": 16.702, "y": 13.398, "target": "GPIO3" },
+    "2/GPIO2/A2": { "x": 16.702, "y": 15.938, "target": "GPIO2" },
+    "1/GPIO1/A1": { "x": 16.702, "y": 18.478, "target": "GPIO1" },
+    "0/GPIO0/A0": { "x": 16.702, "y": 21.018, "target": "GPIO0" },
+    "5/GPIO5/A5/MISO": { "x": 1.462, "y": 3.238, "target": "GPIO5" },
+    "6/GPIO6/MOSI": { "x": 1.462, "y": 5.778, "target": "GPIO6" },
+    "7/GPIO7/SS": { "x": 1.462, "y": 8.318, "target": "GPIO7" },
+    "8/GPIO8/SDA": { "x": 1.462, "y": 10.858, "target": "GPIO8" },
+    "9/GPIO9/SCL": { "x": 1.462, "y": 13.398, "target": "GPIO9" },
+    "10/GPIO10": { "x": 1.462, "y": 15.938, "target": "GPIO10" },
+    "20/GPIO20/RX": { "x": 1.462, "y": 18.478, "target": "GPIO20" },
+    "21/GPIO21/TX": { "x": 1.462, "y": 21.018, "target": "GPIO21" }
+  },
+  "leds": [
+    {
+      "id": "power",
+      "x": 3.121,
+      "y": 7.022,
+      "type": "0603",
+      "color": "red",
+      "pins": {
+        "A": "3V3",
+        "C": "GND"
+      }
+    },
+    {
+      "id": "io8led",
+      "x": 13.599,
+      "y": 12.885,
+      "type": "0603",
+      "color": "blue",
+      "radius": 3.0,
+      "pins": {
+        "A": "8",
+        "C": "GND"      
+      }
+    }
```

Comment:
The LED doesn't work - but it might work after changing the pin names, as suggested above

Reply:
I'll fix it when I get a response on the first one as they are depended on each other :)

Reply:
it's fixed